pkg/services/dns/dns_config_unix.go

```diff
@@ -0,0 +1,22 @@
+//go:build !windows
+
+package dns
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/miekg/dns"
+)
+
+func GetDNSHostAndPort() (string, string, error) {
+	conf, err := dns.ClientConfigFromFile("/etc/resolv.conf")
+	if err != nil {
+		fmt.Fprintln(os.Stderr, err)
+		return "", "", err
+	}
+	// TODO: use all configured nameservers, instead just first one
+	nameserver := conf.Servers[0]
```

Comment:
can we create an issue for this?

Reply:
I created https://github.com/containers/gvisor-tap-vsock/issues/398